modules/@apostrophecms/image/ui/apos/components/AposMediaManagerDisplay.vue

```diff
@@ -246,32 +246,38 @@ export default {
   .apos-media-manager-display__cell {
     @include apos-transition();
 
-    position: relative;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    width: 100%;
-    height: 100%;
+    & {
+      position: relative;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: 100%;
+      height: 100%;
+    }
 
     &.apos-is-hidden { visibility: hidden; }
 
     &:hover,
     &.apos-is-selected,
     &:focus {
       .apos-media-manager-display__media {
-        opacity: 1;
+        & {
```

Comment:
there is no `@include`, it should not be required to use `& {`

Reply:
Refactored.